build.gradle

```diff
@@ -287,10 +292,7 @@ dependencyManagement {
 configurations.all {
     resolutionStrategy {
         eachDependency { DependencyResolveDetails details ->
-            if (details.requested.group in ['com.fasterxml.jackson.core', 'com.fasterxml.jackson.module', 'com.fasterxml.jackson.datatype']) {
-                details.useVersion '2.9.10'
-            }
-            if (details.requested.name == 'jackson-databind') {
+            if (details.requested.group in ['com.fasterxml.jackson.module', 'com.fasterxml.jackson.datatype']) {
```

Comment:
do we still need this config..? May be bump jackson.datatype ?

Reply:
Good spot, I have removed this whole section.